.github/renovate.json

```diff
@@ -0,0 +1,5 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "schedule": "0 0 * * 1",
```

Comment:
From https://docs.renovatebot.com/configuration-options/#schedule
> Warning
> For Cron schedules, you must use the * wildcard for the minutes value, as Renovate doesn't support minute granularity.

Reply:
At least _one_ of us is reading the docs 😆